config/delta-producer/mandatarissen/export.json

```diff
@@ -41,15 +41,8 @@
     },
     {
       "type": "http://data.vlaanderen.be/ns/mandaat#Mandataris",
-      "graphsFilter": [ "http://mu.semte.ch/graphs/organizations/.*/LoketLB-mandaatGebruiker" ],
-      "hasRegexGraphsFilter": true,
-      "pathToConceptScheme": [
-        "http://www.w3.org/ns/org#holds",
-        "^http://www.w3.org/ns/org#hasPost",
-        "http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan",
-        "http://data.vlaanderen.be/ns/besluit#classificatie",
-        "http://www.w3.org/2004/02/skos/core#inScheme"
-      ],
+      "graphsFilter": [ "http://mu.semte.ch/graphs/mandaten" ],
```

Comment:
Just checking ⌛  should this fraph be the same as our `upsert-ldes-entity` **target graph**?
https://github.com/lblod/app-digitaal-loket/pull/565/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R690

Reply:
that should ok